src/Eccube/Controller/Admin/Store/OwnerStoreController.php

```diff
@@ -259,13 +269,33 @@ public function apiInstall(Application $app, Request $request, $pluginCode, $ecc
         }
         $packageNames .= self::$vendorName.'/'.$pluginCode;
         $return = $this->composerService->execRequire($packageNames);
+
         if ($return) {
+            $url = $this->appConfig['package_repo_url'].'/collect';
+            // $composerMode = 1 is ComposerApi
+            $composerMode = 1;
+            if ($this->composerService instanceof ComposerProcessService) {
+                $composerMode = 2;
+            }
+            $data = array(
+                'code' => $pluginCode,
+                'version' => $version,
+                'download_date' => Carbon::now()->format('Y-m-d H:i:s'),
+                'core_version' => $eccubeVersion,
+                'php_version' => phpversion(),
+                'db_version' => $this->systemService->getDbversion(),
+                'os' => php_uname('s'),
+                'host' => php_uname('n') .' '. $request->server->get("REMOTE_ADDR"),
+                'web_server' => $request->server->get("SERVER_SOFTWARE"),
+                'composer_version' => $this->systemService->composerVersion(),
+                'composer_execute_mode' => $composerMode,
+                'dependents' => json_encode($dependents)
```

Comment:
1. please post package name + version only (Not posting all data in `dependents`, it's too much.)
2. Maybe you need to check only the plugin of ec-cube. (please thinking about this).